swan-lake/learn/quick-tour.md

```diff
@@ -12,14 +12,19 @@ redirect_from:
 
 ## Installing Ballerina
 
-1. [Download](https://ballerina.io/downloads) Ballerina based on the Operating System you are using. 
-1. Follow the instructions given on the [Getting Started](/swan-lake/learn/getting-started) page to set it up. 
-1. Follow the instructions given on the [Visual Studio Code Plugin](/swan-lake/learn/tools-ides/vscode-plugin) page or the [IntelliJ IDEA Plugin](/swan-lake/learn/tools-ides/intellij-plugin) page to set up your preferred editor for Ballerina.
+1. [Download](/downloads) Ballerina based on the Operating System you are using. 
+1. [Install](/swan-lake/learn/installing-ballerina) Ballerina. 
+1. Set up [Visual Studio Code](/swan-lake/learn/tools-ides/vscode-plugin) for Ballerina.
 
-## Writing a Service, Running It, and Invoking It
+## Creating a Service and a Client to Invoke It
 
-Write a simple Hello World HTTP service in a file with the `.bal` extension.
+You can use a Ballerina HTTP client to interact with any HTTP service. Let's create a custom Ballerina HTTP service and also an HTTP client to invoke it.
 
+### Writing a simple 'hello' Service
```

Comment:
Use the backtick instead of a single quote.

Reply:
Back tick styles are not supported by the TOC plugin. Therefore, we have to use single quotes for titles.

Reply:
This is how it is displayed in the TOC when we use backticks.

<img width="765" alt="Screenshot 2020-11-16 at 13 13 44" src="https://user-images.githubusercontent.com/11707273/99225214-b1f15080-280d-11eb-96d7-464d50aa0e55.png">

Reply:
What is the issue of using backtick in TOC? I didn't notice the issue in the above image TOC

Reply:
The word with in the backticks (e.g.,`hello`) doesnt get highlighted in the TOC elements.